entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/model/impl/OrganizationImpl.java

```diff
@@ -484,11 +483,6 @@ public void setDcIdentifier(Map<String, List<String>> dcIdentifier) {
 
 		return tmpIdentifier;
 	}
-
-	@Override
-	public String getType() {
-		return "Organization";
```

Comment:
please revert back this code, and use EntityTypes.Organization.getEntityType() instead of the hardcoded value

Reply:
Done.